app/vendor/ToolsAPI.js

```diff
@@ -219,6 +219,24 @@ sap.ui.define(['jquery.sap.global', 'sap/ui/core/library', 'sap/ui/Global', 'sap
                 return result;
             },
 
+            /**
```

Comment:
This information is currently under the   "// Control Properties Info" comment.
It should be under a new commented section called "// Control Aggregations Info"